templates/enter2fa.html

```diff
@@ -73,6 +75,24 @@ <h2>Confirm you're you!</h2>
     </center>
 
 </main>
+<script nonce="{{.NonceCode}}">
+  const otpHandler = async function() {
+    const dataLength = $('#otp').val().length;
+    if(dataLength > 0) {
+        $("#error").hide();
+    }
+    if (dataLength == 6) {
+        await validateOneTimePass(false, $('#otp').val());
+        $('#otp').val("");
+    } else {
+      $("#error").show();
+    }
+  }
+  
+  document.getElementById('otp-button').addEventListener('click', otpHandler);
+  
+  document.getElementById('otp-button').addEventListener('touchstart', otpHandler);
```

Comment:
Do you think this could be moved to templates/assets/script.js, in order to be with the rest of the Javascript stuff?

Reply:
Yes I will try moving them.

Reply:
Moved